lib/v2/components/text_form_field_custom.dart

```diff
@@ -77,11 +79,11 @@ class TextFormFieldCustom extends StatelessWidget {
             color: Colors.red,
             wordSpacing: 4.0,
           ),
-          labelStyle: TextStyle(
-            color: disabledLabelColor == true ? Colors.grey : AppColors.white,
-            letterSpacing: 1.3,
-          ),
-          hintStyle: Theme.of(context).textTheme.subtitle2,
+          labelStyle: Theme.of(context).textTheme.subtitle3.copyWith(
+                color: disabledLabelColor == true ? Colors.grey : AppColors.white,
+                letterSpacing: 1.3,
```

Comment:
why is this needed? We should not be making copies of the styles. 
If the style is not present, then either Design needs to create a new style and we add it to our styles or design needs to change 

Please push back with Jason

Reply:
`letterSpacing: 1.3`.      this is what i am worried about.

Reply:
Condition the color because I thought I did not have a default color for disabled I will remove it like the spacing between letters I see is not necessary.
I'm just waiting for Jason to confirm the color will have the label when the input is selected because I doubt it is the current neon.
![comparation](https://user-images.githubusercontent.com/42857405/111669183-24823500-87dc-11eb-9503-96164c6178a5.png)